README.md

```diff
@@ -116,23 +116,24 @@ Example:
-  | access_token | The access token generated                                          | eyJraWQiOiIwIiwidHlwIjoiYXQrand0IiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiAiZG9tYWluLnRyYXZlbC50cmF2ZWwtc2l0ZSIsImlhdCI6IDE1ODMxMTIwMTYsImV4cCI6IDE1ODMxMTIwMTYsImlzcyI6ICJodHRwczovL3d3dy5hdGhlbnouY29tOjQ0NDMvenRzL3YxIiwiYXVkIjogImRvbWFpbi5zaG9wcGluZyIsImF1dGhfdGltZSI6IDE1ODMxMTIxMTYsInZlciI6IDEsInNjcCI6IFsidXNlciJdLCJ1aWQiOiAiZG9tYWluLnRyYXZlbC50cmF2ZWwtc2l0ZSIsImNsaWVudF9pZCI6ICJkb21haW4udHJhdmVsLnRyYXZlbC1zaXRlIn0.\[signature] |
-  | token_type   | The token type of the access token                                  | Bearer                                                                                                                                                                                                                                                                                                                                                                                                                                 |
-  | expires_in   | The expiry time of the access token                                 | 1528860825                                                                                                                                                                                                                                                                                                                                                                                                                             |
-  | scope        | The scope of the access token (Only added if role is not specified) | user,                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+
+| Name | Description | Example |
+| ---- | ----------- | ------- |
+| access_token | The access token generated | eyJraWQiOiIwIiwidHlwIjoiYXQrand0IiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiJkb21haW4udHJhdmVsLnRyYXZlbC1zaXRlIiwiaWF0IjoxNTgzNzE0NzA0LCJleHAiOjE1ODM3MTY1MDQsImlzcyI6Imh0dHBzOi8venRzLmF0aGVuei5pbyIsImF1ZCI6ImRvbWFpbi5zaG9wcGluZyIsImF1dGhfdGltZSI6MTU4MzcxNDcwNCwidmVyIjoxLCJzY3AiOlsidXNlcnMiXSwidWlkIjoiZG9tYWluLnRyYXZlbC50cmF2ZWwtc2l0ZSIsImNsaWVudF9pZCI6ImRvbWFpbi50cmF2ZWwudHJhdmVsLXNpdGUifQ.\[signature] |
+| token_type   | The token type of the access token | Bearer |
+| expires_in   | The expiry duration of the access token | 1000 |
```

Comment:
better to specify second?

Reply:
fixed: https://github.com/yahoojapan/athenz-client-sidecar/pull/39/commits/954e3400fbfcf07cc972f556e49abd6af1cbbf7e